### PR TITLE
fix: minor bugs

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,9 +16,6 @@ jobs:
         email: $NPM_EMAIL
         api_key: $NPM_API_KEY
         tag: next
-        skip_cleanup: true
-        on:
-          tags: true
```
Comment (stnguyen90) [inline .travis.yml:21]: @loks0n why was this added in https://github.com/appwrite/sdk-for-node/pull/83/commits/1fb9c75ba5fdf9defa3927b1dc0e683f27bd40d8?

Comment (loks0n) [reply .travis.yml:21]: I had assumed it was part of your uppper/lowercase rc version changes I believe. Don't think it's neccessary

**Review by loks0n — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (stnguyen90) [reply .travis.yml:21]: We should be fine without this because the if condition for the stage already checks if the tag has RC or rc in it.

File: src/enums/credit-card.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ export enum CreditCard {
     AmericanExpress = 'amex',
     Argencard = 'argencard',
     Cabal = 'cabal',
-    Consosud = 'censosud',
+    Cencosud = 'cencosud',
```
Comment (TorstenDittmann) [inline src/enums/credit-card.ts:5]: breaking change, but a valid version bump 👍🏻

File: src/enums/name.ts [hunk 3]
```diff
@@ -11,5 +11,4 @@ export enum Name {
     V1builds = 'v1-builds',
     V1messaging = 'v1-messaging',
     V1migrations = 'v1-migrations',
-    Hamsterv1 = 'hamsterv1',
```
Comment (TorstenDittmann) [inline src/enums/name.ts:14]: also breaking change

File: src/services/account.ts [hunk 4]
```diff
@@ -334,9 +334,9 @@ This endpoint can also be used to convert an anonymous account to a normal one,
      * @param {AuthenticatorType} type
      * @param {string} otp
      * @throws {AppwriteException}
-     * @returns {Promise<Models.User<Preferences>>}
+     * @returns {Promise<{}>}
      */
-    async deleteMfaAuthenticator<Preferences extends Models.Preferences>(type: AuthenticatorType, otp: string): Promise<Models.User<Preferences>> {
+    async deleteMfaAuthenticator(type: AuthenticatorType, otp: string): Promise<{}> {
```
Comment (TorstenDittmann) [inline src/services/account.ts:339]: also breaking change

File: src/services/messaging.ts [hunk 5]
```diff
@@ -986,14 +990,14 @@ export class Messaging {
...
-     * @param {string} from
+     * @param {string} templateId
      * @param {string} senderId
      * @param {string} authKey
      * @param {boolean} enabled
      * @throws {AppwriteException}
      * @returns {Promise<Models.Provider>}
      */
-    async createMsg91Provider(providerId: string, name: string, from?: string, senderId?: string, authKey?: string, enabled?: boolean): Promise<Models.Provider> {
+    async createMsg91Provider(providerId: string, name: string, templateId?: string, senderId?: string, authKey?: string, enabled?: boolean): Promise<Models.Provider> {
```
Comment (TorstenDittmann) [inline src/services/messaging.ts:1000]: Also breaking change

File: src/services/messaging.ts [hunk 6]
```diff
@@ -1041,13 +1045,13 @@ export class Messaging {
...
      * @param {boolean} enabled
+     * @param {string} templateId
      * @param {string} senderId
      * @param {string} authKey
-     * @param {string} from
      * @throws {AppwriteException}
      * @returns {Promise<Models.Provider>}
      */
-    async updateMsg91Provider(providerId: string, name?: string, enabled?: boolean, senderId?: string, authKey?: string, from?: string): Promise<Models.Provider> {
+    async updateMsg91Provider(providerId: string, name?: string, enabled?: boolean, templateId?: string, senderId?: string, authKey?: string): Promise<Models.Provider> {
```
Comment (TorstenDittmann) [inline src/services/messaging.ts:1054]: breaking change

**Review by TorstenDittmann — APPROVED** [review]
lgtm, just mentioned some breaking changes for the changelog 👍🏻
